### PR TITLE
feat: add tab for resigned delegates

### LINKED ISSUE #769: Feature: show resigned delegates
The delegate monitor currently shows active and standby delegates, but should also be able to show resigned ones (e.g. with the use of https://dexplorer.ark.io/api/v2/delegates?type=resigned )

### LINKED ISSUE #771: Docs: backwards compatibility
Check backwards compatibility of the new explorer and core 2.5, documenting any breaking changes.

### PR DESCRIPTION
**Pull request by dated**
## Summary

Closes #769.

The added test passes only on devnet, the endpoint itself is currently broken in core, see https://github.com/ArkEcosystem/core/issues/3184 for reference.

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

**Comment by ghost** [comment]
**Thanks for submitting this pull request!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

**Comment by ItsANameToo** [comment]
@dated test failure

**Comment by ItsANameToo** [comment]
> The added test passes only on devnet

ah :trollface:

**Comment by ItsANameToo** [comment]
One thing I'm unsure about is this:
image

The label is meant to be shown in the `active` delegate list when a delegate has resigned during the round. When we view the `resigned` list, it seems redundant information to me to also show the label 🤔

**Comment by dated** [comment]
Totally! I'll change it accordingly.

**Comment by ItsANameToo** [comment]
Will look at the PR again when devnet gets updated and the delegate ranking issue is fixed. Should be soon-ish (in the next few days)

**Comment by dated** [comment]
No worries. What are your thoughts on adding pagination to the standby and resigned tables?

**Comment by ItsANameToo** [comment]
🤔 would make sense to have pagination there too

**Comment by ItsANameToo** [comment]
can you use `store.dispatch("network/setServer", "https://dexplorer.ark.io/api");` for now in the "should retrieve the resigned delegates" unit test @dated ? So we can have that test pass

**Review by ItsANameToo — APPROVED** [review]

**Comment by ghost** [comment]
A member has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ghost** [comment]
Your pull request has been merged and marked as tier 4. _It will earn you $20 USD._

File: resources/views/livewire/stats-highlights.blade.php [hunk 1]
```diff
@@ -1,6 +1,23 @@
-<div wire:poll.{{ $refreshInterval }}s class="flex gap-5 w-full md:grid md:grid-cols-2 md:grid-rows-2 xl:grid-cols-4 xl:grid-rows-1">
-    <x-stats.highlight icon="stacked-coins" :title="trans('pages.statistics.highlights.total-supply')" :value="$totalSupply" />
```
Comment (ItsANameToo) [inline resources/views/livewire/stats-highlights.blade.php:2]: is `stats.highlight` still used? if not: delete it

File: resources/views/livewire/stats-highlights.blade.php [hunk 2]
```diff
@@ -3,11 +3,11 @@
         <div class="px-10 md:w-full">
             <div wire:poll.{{ $refreshInterval }}s class="flex gap-3 w-full md:grid md:grid-cols-2 md:grid-rows-2 xl:grid-cols-4 xl:grid-rows-1">
                 <x-stats.stat :label="trans('pages.statistics.highlights.total-supply')" icon="stacked-coins">
-                    <x-currency :currency="Network::currency()">{{ $totalSupply }}</x-currency>
+                    {{ $totalSupply }}
```
Comment (ItsANameToo) [inline resources/views/livewire/stats-highlights.blade.php]: wouldn't it be better to keep the `x-currency` and remove the currency from being added in the livewire component? Then the total supply is still a normal number and the frontend decides how to show it

**Pull request merged**